docs/sql/statements/copy-from.rst

```diff
@@ -287,6 +287,37 @@ Using the ``s3://`` scheme automatically sets the `shared`_ to true.
    0.51.x these connections are using the HTTPS protocol. Please make sure you
    update your firewall rules to allow outgoing connections on port ``443``.
 
+.. _sql-copy-from-azblob:
+
+``azblob``
+''''''''''
+
+You can use the ``azblob://`` scheme to access files on the `Azure Blob Storage`_.
+
+URI must consist of scheme and absolute path to the resource.
+
+For example:
+
+.. code-block:: text
+
+    azblob://dir1/dir2/file1.json
```

Comment:
A valid URI without any host component must use 3 slashes.
```suggestion
    azblob:///dir1/dir2/file1.json
```

Reply:
I aligned with s3 https://cratedb.com/docs/crate/reference/en/latest/sql/statements/copy-from.html#s3

Will update and also will check if it helps to get rid of `AzureFileOutput.resourcePath` and use `uri.getPath()`.